VR metadata scene

joseph-kaile opened the pull request:
This sample scene adds metadata picking for VR using the XR interaction toolkit. It also enables some performance improvement settings for the Quest 2. 

It also makes improvements to the Denver scene.

Also added files from XR interaction Samples and the Device simulator.

Utilizes pull request https://github.com/CesiumGS/cesium-unity/pull/336.

kring commented:
> I thought we discussed making a new repository for the VR samples at some point. Is there a reason we didn't go through with that?

Only that it increases our maintenance burden a bit (another project to maintain and release), and the benefit (being able to default to the built-in render pipeline) didn't quite seem to justify it. I'm not opposed to going that way if it seems worthwhile, though.

j9liu commented on Assets/CesiumForUnitySamples/Scripts/CesiumSamplesMetadataPickingVR.cs:
```diff
@@ -0,0 +1,137 @@
+                    var mc = child.GetComponent<MeshCollider>();
+                    if (mc != null)
+                    {
+                        Physics.IgnoreCollision(mc, characterController);
```
Is this necessary for the level to work? Or was this just a decision to let the player pass through buildings?

joseph-kaile replied:
Yes, you would start walking on top of the smaller buildings. Also I thought it was neat to demonstrate customizing the game objects.

j9liu reviewed (commented):
@joseph-kaile I tried this out on the Quest 2 and it looks great! I had some comments about usability and folder organization. Let me know if you have any questions

j9liu commented on Assets/CesiumForUnitySamples/Scenes/VR02_CesiumMetadata.unity:
```diff
@@ -0,0 +1,2874 @@
+%YAML 1.1
+%TAG !u! tag:unity3d.com,2011:
```
One more thing -- the "Getting Started" text is the exact same as the first VR level. Since we already have build instructions for the first level, let's have the second level's description be more specific about what it's showing. Feel free to reference the other scenes' text for inspiration.

We also need a section for this in the README.

joseph-kaile commented:
> @joseph-kaile I tried this out on the Quest 2 and it looks great! I had some comments about usability and folder organization. Let me know if you have any questions

Thanks for the review. I implemented all the suggestions and also updated the scene description and readme.
